README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
I agree that's the crux of the problem. I think this is one of those situations where more words is better. It would be great to spell out why there are two different packages, and go into detail about who should use each one. It's frustrating to have a confusing/ambiguous decision point right at the very beginning of one's relationship with a library.